backend/pyproject.toml

```diff
@@ -1,48 +1,5 @@
-[tool.poetry]
-name = "flathub-backend"
-version = "0.1.0"
-description = "Backend for Flathub website"
-authors = ["Bartłomiej Piotrowski <bpiotrowski@gnome.org>"]
-license = "Apache-2.0"
-
-[tool.poetry.dependencies]
-python = "^3.11"
-fastapi = "^0.96.0"
-uvicorn = "^0.22.0"
-sentry-sdk = "^1.25.0"
-lxml = "^4.9.2"
-requests = "^2.31.0"
-redis = "^4.5.5"
-feedgen = "^0.9.0"
-PyGObject = "^3.44.1"
-watchgod = "^0.8"
-FastAPI-SQLAlchemy = "^0.2.1"
-alembic = "^1.11.1"
-psycopg2-binary = "^2.9.6"
-itsdangerous = "^2.1"
-PyGithub = "^1.58"
```

Comment:
It also seems way easier to be able to figure out which version is installed/targeted. With the new tooling, you would need to look into the requirement file, to even get a rough version.

I guess there would be a cli command, but that doesn't help you if you're looking at it on github.

Reply:
I mean, you can just look into requirements.txt to see what it has been resolved to. It doesn't functionally change anything, apart from the dependences being defined in requirements.in. Managing deps in pyproject.toml is also supported.